macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -436,34 +452,69 @@ fileprivate extension Unicode.Scalar {
 
 fileprivate extension NSColor {
 	/// the text color of the text in the initials view
-	///
-	/// - note: this value doesn't change for dark mode by design as our default background color table
-	/// only provides colors designed to be used with white text, even on dark mode
-    static let initialsViewTextColor: NSColor = .white
 
 	/// the table of background colors for the initials views
-	static let avatarBackgroundColors: [NSColor] = [
-		Colors.Palette.cyanBlue10.color,
-		Colors.Palette.red10.color,
-		Colors.Palette.magenta20.color,
-		Colors.Palette.green10.color,
-		Colors.Palette.magentaPink10.color,
-		Colors.Palette.cyanBlue20.color,
-		Colors.Palette.orange20.color,
-		Colors.Palette.cyan20.color,
-		Colors.Palette.orangeYellow20.color,
-		Colors.Palette.red20.color,
-		Colors.Palette.blue10.color,
-		Colors.Palette.magenta10.color,
-		Colors.Palette.gray40.color,
-		Colors.Palette.green20.color,
-		Colors.Palette.blueMagenta20.color,
-		Colors.Palette.pinkRed10.color,
-		Colors.Palette.gray30.color,
-		Colors.Palette.blueMagenta30.color,
-		Colors.Palette.gray20.color,
-		Colors.Palette.cyan30.color,
-		Colors.Palette.orange30.color
+	static let avatarColors: [ColorSet] = [
+		ColorSet(background: NSColor.getColor(light: Colors.Palette.darkRedTint40.color, dark: Colors.Palette.darkRedShade30.color),
+				 foreground: NSColor.getColor(light: Colors.Palette.darkRedShade30.color, dark: Colors.Palette.darkRedTint40.color)),
```

Comment:
nit: is there a line break for the 2 params? They look misaligned on the GitHub web interface for the PR.

Reply:
That seems to be the github interface, I have attached snapshot for XCode. 

![Screen Shot 2021-03-01 at 4 50 50 PM](https://user-images.githubusercontent.com/63682282/109579739-5726f080-7aae-11eb-8d0f-5acac63179a6.png)

I believe we don't have line character limit in Xcode or Swiftlint so this might look different on your end. However, there is no intentional line break added in the code

Reply:
I think Github thinks tabs are 8 spaces by default. You can fix that by adding a .editorconfig file

Reply:
editorconfig? This is for github editor? If there is an example, I can add it to the repo.

Reply:
I have created a new github issue to track the addition of editorconfig file